### PR TITLE
docs: Add biweekly-report-0621

### PR DESCRIPTION
**Pull request by Qizeng-api7**
The content of this PR is the biweekly community report（0606-0618）.

### CODE REVIEW

File: blog/en/blog/2023/06/21/weekly-report-en.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+Link: https://github.com/apache/apisix/issues/9676
+
+Description: Add Chinese documentation for the loki plugin
+
+### Issue #9691
+
+Link: https://github.com/apache/apisix/issues/9691
+
+Description: Add custom header forwarding capability for the proxy-mirror
+
```
Comment (Yilialinn) [inline blog/en/blog/2023/06/21/weekly-report-en.md]: Delete this part. These are already assigned.

File: blog/en/blog/2023/06/21/weekly-report-en.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+## Highlights of Recent Features
```
Comment (Yilialinn) [inline blog/en/blog/2023/06/21/weekly-report-en.md]: Only one, no "s"

**Review by Yilialinn — COMMENTED** [review]
Revise the Chinese version accordingly

**Comment by Yilialinn** [comment]
> 

@Qizeng-api7 Fulfill this part every time and also add dates on the title for distinguishing it from other blogs!
image

File: blog/zh/blog/2023/06/21/weekly-report-zh.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+![贡献者名单](https://static.apiseven.com/uploads/2023/06/20/Z6yEOnRT_%E5%90%8D%E5%8D%95.png)
+
+![新晋贡献者](https://static.apiseven.com/uploads/2023/06/20/zvIq9JId_%E6%96%B0%E6%99%8B%E8%B4%A1%E7%8C%AE%E8%80%85%E6%B5%B7%E6%8A%A5.png)
+
+## 近期亮点功能
+
+### Apache APISIX
+
+- [确保 pb 状态可以被正确恢复](https://github.com/apache/apisix/pull/9606)（贡献者：[kingluo](https://github.com/kingluo)）
```
Comment (Yilialinn) [inline blog/zh/blog/2023/06/21/weekly-report-zh.md]:
```suggestion
- [重新编译 protobuffer 时，确保正确重载旧状态](https://github.com/apache/apisix/pull/9606)（贡献者：[kingluo](https://github.com/kingluo)）
```

**Review by Yilialinn — APPROVED** [review]

**Review by guoqqqi — APPROVED** [review]

**Review by SkyeYoung — APPROVED** [review]

**Comment by yzeng25** [comment]
@Yilialinn @Qizeng-api7 Could you follow previous pull requests to update both file names in another pull request?

`blog/zh/blog/2023/06/21/weekly-report-zh.md` should be `blog/zh/blog/2023/06/21/weekly-report.md`;

`blog/en/blog/2023/06/21/weekly-report-en.md` should be `blog/en/blog/2023/06/21/weekly-report.md`.

**Pull request merged**